civis/civis-aqueduct-utils/civis_aqueduct_utils/share.py

```diff
@@ -67,3 +70,41 @@ def main():
 
 if __name__ == "__main__":
     main()
+
+
+# Function to load file onto GitHub
+# Works with CSVs, HTML, but not parquet?
+def upload_file_to_github(TOKEN, REPO, BRANCH, PATH, local_file_path, commit_message):
```

Comment:
@ian-r-rose Are there more parameters to be added? Parquet files don't work with encoding to text file so far.

Reply:
These parameters look good. Can we lowercase `TOKEN`, `REPO`, `BRANCH`, and `PATH`? The convention I follow is usually to lower-case parameters and variables, and upper-case constants.